docs/manual/manual_stow.rst

```diff
@@ -219,12 +219,61 @@ Refer to the :ref:`timeout_behavior` section for more information on how to conf
 
 Holding Position
 ==================
-Once the module reaches the stow position, it can either continue actively trying to hold that position or it can coast the motor and let it spin freely. This 
-behavior is controlled by the *Hold Stow* parameter, as mentioned in the :ref:`stow_parameter_summary` section above.
+Once the module reaches the stow position, it can transition to one of several different behaviors. Which behavior is controlled by the *Hold Stow* parameter, as mentioned in the :ref:`stow_parameter_summary` section above.

```

Comment:
, it can transition to one of several different behaviors as defined by the *Hold Stow* parameter. You can read more about this above <link>.